content/v2.0/query-data/flux/manipulate-timestamps.md

```diff
@@ -26,13 +28,21 @@ Use Flux to process and manipulate timestamps to suit your needs.
 - [Convert timestamp format](#convert-timestamp-format)
 - [Time-related Flux functions](#time-related-flux-functions)
 
+---
```

Comment:
Think the lines work well in longer topics. Here, it's hard to tell what the break implies--maybe the note format?

Also see this "note" used in many topics, so we'll want to search and replace across the board for consistency.